app/src/main/java/simararora/puautologin/LoginTask.java

```diff
@@ -21,7 +21,7 @@
 public class LoginTask extends AsyncTask<Void, String, Void> {
 
     private Context context;
-    private static final String loginURL = "https://securelogin.arubanetworks.com/cgi-bin/login?cmd=login";
+    private static final String loginURL = "https://securelogin.pu.ac.in/cgi-bin/login?cmd=login";
```

Comment:
@s-gumber Can you send me the source code of the login page. My email id is simarpreetsingharora@gmail.com